issue-wanted.cabal

```diff
@@ -34,23 +34,82 @@ common common-options
+  default-extensions:  BangPatterns
+                       ConstraintKinds
+                       DataKinds
+                       DeriveGeneric
+                       DerivingStrategies
+                       DerivingVia
+                       FlexibleContexts
+                       FlexibleInstances
                        GeneralizedNewtypeDeriving
+                       InstanceSigs
+                       KindSignatures
                        LambdaCase
+                       MultiParamTypeClasses
                        OverloadedStrings
+                       OverloadedLabels
                        RecordWildCards
                        ScopedTypeVariables
                        TypeApplications
+                       TypeFamilies
+                       TypeOperators
 
 library
   import:              common-options
   hs-source-dirs:      src
   exposed-modules:     Prelude
+
                        IW
-                       IW.Search
-  build-depends:       github == 0.20 
+                         -- Application Monad
+                         IW.App
+                           IW.App.Env
+                           IW.App.Error
+                           IW.App.Monad
+                        
+                         IW.Config
+                         
+                         -- Database
+                         IW.Db
+                           IW.Db.Functions
+                           IW.Db.Schema
+                      
+                         -- Effects
+                         IW.Effects.Log  
+                         
+                         -- Server
+                         IW.Server
+                           IW.Server.Search
+  build-depends:       aeson >= 1.4
+                     , bcrypt ^>= 0.0.11
+                     , bytestring ^>= 0.10
+                     , case-insensitive ^>= 1.2
+                     , co-log ^>= 0.3.0
+                     , containers >= 0.6
+                     , ekg ^>= 0.4.0.15
+                     , ekg-core ^>= 0.1.1.6
+                     , github == 0.20
+                     , http-api-data ^>= 0.4
+                     , http-types ^>= 0.12
+                     , jwt ^>= 0.9.0
+                     , lens ^>= 4.17
+                     , mtl ^>= 2.2.2
+                     , postgresql-simple ^>= 0.6.1
+                     , proto-lens ^>= 0.5.0.0
+                     , proto-lens-protoc ^>= 0.5.0.0
+                     , proto-lens-runtime ^>= 0.5.0.0
+                     , random ^>= 1.1
+                     , resource-pool
+                     , servant ^>= 0.16
+                     , servant-proto-lens ^>= 0.1.0.3
+                     , servant-server ^>= 0.16
+                     , text
+                     , time >= 1.8 && < 1.10
+                     , tomland ^>= 1.0.0
+                     , unordered-containers
+                     , warp ^>= 3.2
 
-executable issue-wanted
+executable issue-wanted-exe
```

Comment:
Is there any reason to add `-exe` suffix here? I think it can be just `issue-wanted` for now (also, don't forget to change `Makefile` after renaming).

Reply:
I got the idea from here: https://github.com/Holmusk/three-layer/blob/eedbed6d1795eefe7d23779ee83ee7e0885bd9b1/three-layer.cabal#L151-L157

I will change it to just `issue-wanted` for now